CHANGELOG_PENDING.md

```diff
@@ -25,6 +25,7 @@ Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermi
   - [rpc/grpc] \#6725 Mark gRPC in the RPC layer as deprecated.
   - [blockchain/v2] \#6730 Fast Sync v2 is deprecated, please use v0
   - [rpc] Add genesis_chunked method to support paginated and parallel fetching of large genesis documents.
+  - [rpc] \#6820 Update RPC methods to reflect changes in the p2p layer, disabling support for `UnsafeDialPeers` and `UnsafeDialPeers` when used with the new p2p layer, and changing the response format of the peer list in `NetInfo` for all users.
```

Comment:
is there anyway we could support adding peers without stopping and starting a node? This will be a major UX regression, IMO, and we already have poor UX. 

We should have running nodes on various networks as the onboarding process.

Reply:
> is there anyway we could support adding peers without stopping and starting a node?

Not really. This isn't consistent with the design of the new p2p stack, and would require some disruptive changes to the router and the peer manager. We could add a method that was like `UnsafeForceAddPeer` but it wouldn't (necessarily) cause that peer to be dialed with any particular haste, and I think it'd be confusing 

>  This will be a major UX regression, IMO, and we already have poor UX.

Arguably Unsafe RPC methods are very poor UX. 

We reached out to the validautor channel, and no one seemed particularly interested in the topic either way. 

I did reach out directly to Greg at informal for his experience, and he left [this comment](https://github.com/tendermint/tendermint/issues/6555#issuecomment-896961116) which indicates an explicit disinterest in the unsafe methods. He also, in our conversation seemed to be ok with restarts in this kind of situation. 

Greg also seems to think that restarts for config changes (particularly when using a management layer like nomad/k8s/etc.) are common and shouldn't be avoided. This is consistent with my own operational experience.